packages/create-cloudflare/src/git.ts

```diff
@@ -26,37 +27,11 @@ export const offerGit = async (ctx: C3Context) => {
 		return; // bail early
 	}
 
-	const insideGitRepo = await isInsideGitRepo(ctx.project.path);
-
-	if (insideGitRepo) {
-		// Store that we're in an existing git repo for later reference
-		ctx.gitRepoAlreadyExisted = true;
-
-		// If git flag was explicitly set to false, respect that
-		if (ctx.args.git === false) {
-			return;
-		}
-
-		// If git flag was explicitly set to true, respect that
-		if (ctx.args.git === true) {
-			return;
-		}
-
-		// Otherwise, ask the user if they want to use git
-		ctx.args.git = await processArgument(ctx.args, "git", {
-			type: "confirm",
-			question:
-				"You're in an existing git repository. Do you want to use git for version control?",
-			label: "git",
-			defaultValue: C3_DEFAULTS.git,
-		});
-
-		return;
-	}
-
-	ctx.args.git = await processArgument(ctx.args, "git", {
+	ctx.args.git ??= await processArgument(ctx.args, "git", {
 		type: "confirm",
-		question: "Do you want to use git for version control?",
+		question: ctx.gitRepoAlreadyExisted
+			? "You're in an existing git repository. Do you want to use git for version control?"
```

Comment:
Comment:

I don't really understand "You're in an existing git repository. Do you want to use git for version control?"

i.e. am I not using git for version control if I am in a git repo?

Reply:
This was added by Dan in the previous PR that caused the regression that this PR is fixing.
The idea is that you might be in some monorepo that is using git, but still not want this particular directory to be tracked via git - and so no git commit should be created.